### DISCUSSION TITLE
Is there any way to enable autologin?

### DISCUSSION THREAD

**Discussion by wetherby** [stringer-rss/stringer#887] [Q&A] [answered]
I'm liking stringer-rss so far but is there any way to enable autologin? I'm self hosting a docker container on my own network and and don't anticipate any requirement for multiple users.

**Comment by mockdeep** [comment, marked as answer]
@wetherby I just merged [a PR to extend sessions to 2 weeks](https://github.com/stringer-rss/stringer/pull/889), so if you pull the latest `main` you should be able to stay logged in longer.
